### PR TITLE
Fossord rebranded to spacebar

### PR DESCRIPTION
**Pull request by erkinalp**
Fosscord has recently rebranded to Spacebar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,7 +160,7 @@ The following are server implementations that reimplement Discord's client-serve
 
 | Name | Features | Language(s) | Development Status |
 | :---: | :---: | :---: | :---: |
-| [Fosscord](https://fosscord.com/) | Almost fully featured re-implementation of Discord API Server, intended for self hosting | [![JavaScript][JavaScript-Badge]][JavaScript-Url] [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🟢 Active |
+| [Spacebar Chat](https://spacebar.chat/) | Almost fully featured re-implementation of Discord API Server, intended for self hosting | [![JavaScript][JavaScript-Badge]][JavaScript-Url] [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🟢 Active |
```
Comment (sehnryr) [inline README.md:163]: We should consider to remove this entry from the server implementations as Spacebar do not advertise their server as it anymore

File: README.md [hunk 2]
```diff
@@ -103,6 +103,7 @@ A non-exhaustive collection of third-party clients and mods for Discord.
 | [RyU](https://github.com/Muunatic/RyU) | Powerful Discord Client written in JavaScript. Lightweight, Efficient, Feature-rich. | [![JavaScript][JavaScript-Badge]][JavaScript-Url] | 🟢 Active |
 | [Swiftcord](https://github.com/cryptoAlgorithm/Swiftcord) | A completely native Discord client for macOS built 100% in Swift and SwiftUI! | [![Swift][Swift-Badge]][Swift-Url] | 🟢 Active |
 | [WebCord](https://github.com/SpacingBat3/WebCord) | A Discord and Fosscord API-less client made with Electron | [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🔵 Active *(Variable[^2]\)* |
+| [Spacebar Chat](https://spacebar.chat/) | Almost fully featured third party Discord API reimplementation, the client | [![JavaScript][JavaScript-Badge]][JavaScript-Url] [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🟢 Active |
```
Comment (sehnryr) [inline README.md:106]:
```suggestion
| [Spacebar Chat](https://spacebar.chat/) | Open source, themeable and extendable discord-compatible native Spacebar client | [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🟢 Active |
```

Spacebar's client repo had its description changed, and they now mostly uses Typescript so the Javascript badge can be removed.
image

File: README.md (hunk 1 above)
Comment (sehnryr) [reply README.md:163]:
```suggestion
```

Comment (erkinalp) [reply README.md:163]: The server still exists.

**Pull request merged**